src/Resources/skeleton/resetPassword/twig_email_txt.tpl.php

```diff
@@ -0,0 +1,9 @@
+Hi!
+
+To reset your password, please copy the below link into your browser.
+
+{{ url('app_reset_password', {token: resetToken.token}) }}
+
+This link will expire in {{ tokenLifetime|date('g') }} hour(s)..
```

Comment:
Super minor comment: there's an extra dot at the end of this phrase.

Reply:
This comes directly from the current template https://github.com/symfony/maker-bundle/blob/bb1438ba9d121f58d40e1d3d5a02d2fea45c8ad4/src/Resources/skeleton/resetPassword/twig_email.tpl.php#L6 I thought it might be intentionally there